sys-block/rts_pstor/metadata.xml

```diff
@@ -1,7 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!--maintainer-needed-->
+	<maintainer type="person">
+		<email>martin.dummer@gmx.net</email>
+		<name>Martin Dummer</name>
+		<description>Proxied maintainer</description>
```

Comment:
Don't state the obvious.

Reply:
You still state the obvious. Remove the useless description.

Reply:
yeah. removed.